netatopif.c

```diff
@@ -149,8 +149,7 @@ netatop_probe(void)
 		{
 			(void) close(netexitfd);
 
-			if (nahp)
-				munmap(nahp, sizeof *nahp);
+			munmap(nahp, sizeof *nahp);
```

Comment:
I don't believe munmap has the same null-ignoring behaviour as free here

Reply:
@DavidBuchanan314, I answered about it here: https://github.com/Atoptool/atop/commit/8d1799bff61461ef151aed6e05b05cacb6475648#commitcomment-154345184